source/data/tier1data/basics/architecture.rst

```diff
@@ -0,0 +1,43 @@
+.. _architecture:
+
+########################
+Tier-1 Data Architecture
+########################
+
+The Tier-1 Data platform is based on the open source software iRODS.
+The image below shows the high level architecture of the platform:
+
+.. figure:: ../images/introduction/tier1data_architecture.png
+   :alt: A schematic representation of the architecture of Tier-1 Data with, from bottom to top: the storage layer, the layer of iRODS middleware, and the different clients users can use.  
+
+An single installation of iRODS is called a 'zone'.
+By default, new projects are creates in our general zone 'VSC'.
+Separate zones can be created for confidentiality or efficiency reasons for different projects. 
+
+Each zone has a Rule engine and an iCAT
+database, which contains metadata, permissions and an index of all file locations.
+Inside a zone, each project has their own folder. 
+
+The data itself is synchronized on two separate hardware storage
+systems, each 27 PB large, located at Leuven and at Heverlee (ICTS KU Leuven). 
+The data is protected against calamities at either site by synchronizing it in real-time at hardware level. 
+One system does not function as a backup for the other, so this is no protection against accidental instructions
+(i.e. user mistakes) to delete data. 
+Snapshots are made at regular intervals (hourly, daily and monthly) in case data needs to be recovered.
+For data recovery in case of emergency, please contact data@vscentrum.be. 
+
+An iRODS zone can be accessed from different systems:
+
+- The HPC clusters of the Flemish Supercomputing Center (VSC)
+- Your laptop
+- Scientific instruments like scanners and microscopes
+- ...
```

Comment:
I'd rather not delete it, since
1) It may otherwise not be obvious that data is accessible from outside the VSC systems
2) Automatic ingestion of data is becoming a big selling point
I tried to rephrase it, pushing soon

Reply:
I'm just talking about the last one with the 3 dots. The rest are fine.

Reply:
Ah, oops